<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhysicsListDialog</class>
 <widget class="QDialog" name="PhysicsListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select physics list &amp; number of threads</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(32, 74, 135);
font: 8pt &quot;Ubuntu Condensed&quot;;
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="open_create">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>create new ...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>open file</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="recent_file_label">
       <property name="text">
        <string>Recent files:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="open_other_files">
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Open other files ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="physicsLists">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="physicsListHint">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="recent_file_tree">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="indentation">
      <number>20</number>
     </property>
     <attribute name="headerMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Physics list</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="firstTimeUse">
       <property name="text">
        <string>First time use</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="disable_vis">
       <property name="text">
        <string>Disable visualization</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of threads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="nThreads">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PhysicsListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PhysicsListDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
